martha_v2/martha_v2.js

```diff
@@ -11,7 +11,9 @@ function parseRequest(req) {
 
 function maybeTalkToBond(req, provider = BondProviders.default) {
     let myPromise;
-    if (req && req.headers && req.headers.authorization) {
+    // Currently HCA data access does not require additional credentials.
+    // The HCA checkout buckets allow object read access for GROUP_All_Users@firecloud.org.
+    if (req && req.headers && req.headers.authorization && provider != BondProviders.HCA) {
```

Comment:
In addition to here, we'll also need a similar change in `fileSummaryV1/service_account_keys.js`

Reply:
Good catch @panentheos, thanks.  @mikebaumann - I think we're looking at around this line: https://github.com/mikebaumann/martha/blob/764ec6a1bb3f39349165b113facb0c2932069297/fileSummaryV1/service_account_keys.js#L5 and doing something similar which is if the provider is `HCA`, then just return a resolved Promise.

Reply:
Thanks, @panentheos, @gpolumbo-broad!
I'm back in the office now and committed the corresponding change for File Summary V1. Please have a look. If you would like the conditional to be structured differently, please let me know.